interface/forms/aftercare_plan/new.php

```diff
@@ -26,153 +28,198 @@
+                    </td>
+                    <td colspan="3">
+                        <input type="text" class="form-control" name="goal_b" value=""
+                               placeholder="<?php echo xlt('Enter patient second goal'); ?>">
+                    </td>
+                </tr>
+                <tr>
+                    <td class="forms">1.</td>
+                    <td colspan="3">
+                        <textarea
+                                name="goal_b_emotional_behavioral_conditions"
+                                class = "form-control"
+                                rows="2" cols="80" wrap="virtual name"><?php
+                                echo text($obj["goal_b_emotional_behavioral_conditions"]);?></textarea>
+                    </td>
+                </tr>
+                <tr>
+                    <td class="forms">2.</td>
+                    <td colspan="3">
+                        <textarea name="goal_b_emotional_behavioral_conditions_I"
+                                  class = "form-control"
+                                  rows="2" cols="80" wrap="virtual name"><?php
+                                    echo text($obj["goal_b_emotional_behavioral_conditions_I"]);?></textarea>
+                    </td>
+                </tr>
+
+                <tr>
+                    <td class="forms-subtitle">
+                        <label for="goal_c"> <strong><?php echo xlt('Goal C');?>:</strong></label>
+                    </td>
+                    <td colspan="3">
+                        <input type="text" class="form-control" name="goal_c" value=""
+                               placeholder="<?php echo xlt('Enter patient third goal'); ?>">
+                    </td>
+                </tr>
+                <tr>
+                    <td class="forms">1.</td>
+                    <td colspan="3">
+                        <textarea name="goal_c_relapse_potential"
+                                  class = "form-control"
+                                  rows="2" cols="80" wrap="virtual name"><?php
+                                    echo text($obj["goal_c_relapse_potential"]);?></textarea>
+                    </td>
+                </tr>
+                <tr>
+                    <td class="forms">2.</td>
+                    <td colspan="3">
+                        <textarea name="goal_c_relapse_potential_I"
+                                  class = "form-control"
+                                  rows="2" cols="80" wrap="virtual name"><?php
+                                    echo text($obj["goal_c_relapse_potential_I"]);?></textarea>
+                    </td>
+
+                </tr>
+                <tr>
+                    <td></td>
+                    <td><input type='submit' value='<?php echo xla('Save'); ?>' class='btn btn-primary' />&nbsp;
+                        <input type='button' value='<?php echo xla('Print'); ?>'
```

Comment:
Convert `input` to `button` and make it `btn-secondary` instead of `btn-warning`